src/constraint/dart/InverseKinematicsSampleable.cpp

```diff
@@ -220,8 +220,8 @@ bool IkSampleGenerator::sample(statespace::StateSpace::State* _state)
 
     mInverseKinematics->getTarget()->setTransform(poseState.getIsometry());
 
-    // Run the IK solver. If it succeeds, return the solution.
-    if (mInverseKinematics->solve(true))
+     // Run the IK solver. If an exact solution is computed, apply it to the skeleton.
+    if (mInverseKinematics->solveAndApply(true))
```

Comment:
It seems AIKIDO still support Ubuntu 14.04. The latest DART version available in Ubuntu 14.04 is 6.6.2. I would change this to be backward compatible something like:
```cpp
#if DART_VERSION_AT_LEAST(6,8,0)
    if (mInverseKinematics->solveAndApply(true))
#else
    if (mInverseKinematics->solve(true))
```